### ISSUE TITLE
How can I add this website to my online dictionaries in Goldendict? 

### ISSUE THREAD

**Issue by nnnfuuu** [goldendict/goldendict#1553]
Hello, is there way to add this website to my online dictionaries in Goldendict? I tried but its not working. This site right here: 

http://medisoft.com.vn/medic.asp

Thanks in advance!

**Comment by xiaoyifang** [comment]
http://medisoft.com.vn/medic.asp?txtSOURCE=%GDWORD%&INPUT=I2&TIMKIEM=TIMKIEM

**Comment by nnnfuuu** [comment]
> http://medisoft.com.vn/medic.asp?txtSOURCE=%GDWORD%&INPUT=I2&TIMKIEM=TIMKIEM

@xiaoyifang I just update GoldenDict to 5.15.2-GoldenDict-v22, using your link like above. Translation is there but the interface looks like this. Do you know the reason? Any way  to fix this? 

Thanks for your help!

![9tZ4zEpIpt](https://user-images.githubusercontent.com/113565214/190478766-60f44eb1-3707-4258-911c-c6cc80885f49.gif)

**Comment by darnn** [comment]
You may want to try the last 2019 builds:
https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Windows

All the basic functionality is there, and I believe the website should at least be displayed as an actual website rather than source code.

Conversely, if you're not satisfied with that, fell free to email me (thedarnn@gmail.com), I may have something else to offer you.

**Comment by nnnfuuu** [comment]
> You may want to try the last 2019 builds: https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Windows
> 
> All the basic functionality is there, and I believe the website should at least be displayed as an actual website rather than source code.
> 
> Conversely, if you're not satisfied with that, fell free to email me ([thedarnn@gmail.com](mailto:thedarnn@gmail.com)), I may have something else to offer you.

@darnn Thanks for your reply! The website only appears the first time i click on text. its not working the second time, as you see in the gif. I tried to quit the app, as well as reinstalling but the same thing still happened. the website wasnt displayed in the second attempt, isnt that weird!

![8txChc9EpM](https://user-images.githubusercontent.com/113565214/190515828-9142794d-27cb-45d8-83bc-19f619e76b0c.gif)

**Comment by darnn** [comment]
@nnnfuuu I agree, it is indeed weird! Unfortunately I don't really know what to do about it in that version. I can however offer you a version where it does seem to work even if you click on a word in your history, but again, you'd have to email me, because discussion of it is not exactly welcome here.
